pkg/tfbridge/diff.go

```diff
@@ -117,8 +117,8 @@ func visitPropertyValue(name, path string, v resource.PropertyValue, tfs *schema
 				// forms part of the key. We round-trip through a config field reader so that TF has the opportunity to
 				// fill in default values for empty fields (note that this is a property of the field reader, not of
 				// the schema) as it does when computing the hash code for a set element.
-				ev, err := MakeTerraformInput(
-					nil, ep, resource.PropertyValue{}, e, etfs, eps, make(AssetTable), nil, false, rawNames)
+				ctx := &conversionContext{}
+				ev, err := ctx.MakeTerraformInput(ep, resource.PropertyValue{}, e, etfs, eps, rawNames)
```

Comment:
Can we please have more descriptive variable names? I know a lot of the above is existing code but I have no idea what etfs, tfs, sch, issch, ps, eps, e, ep, ti or ev are.

Reply:
I won’t change the variable names with this PR, but we can elaborate them in a stand-alone change.